pom.xml

```diff
@@ -17,12 +17,12 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-   
+
     <parent>
         <groupId>org.eclipse.microprofile</groupId>
         <artifactId>microprofile-parent</artifactId>
-        <version>2.6</version>
-    </parent> 
+        <version>3.1</version>
```

Comment:
```suggestion
        <version>3.2</version>
```

Reply:
Thank you, sorry I missed that. I've updated #368. We can either go with #368 first or I can combine them here. I'm okay either way.

Reply:
I approved the PR #368 . I think we can merge that one first and then you can rebase. What do you think?

Reply:
+1, once that is merged I'll get this one updated and it will only contain the workflow file.

Reply:
#368 was merged.

Reply:
Thank you! I've updated this commit and made it a PR.